### DISCUSSION TITLE
Checking for updates of Freeplane?

### DISCUSSION THREAD

**Discussion by wordmuse** [freeplane/freeplane#307] [Help] [answered]
Where are the installation files located for Freeplane updates?  Will the process for alpha versions, stable versions, and archiving older versions be similar to what they were on SourceForge?

Thanks.

~ Bal

**Comment by macmarrum** [comment, marked as answer]
New releases of Freeplane are still available on [SorceForge](https://sourceforge.net/projects/freeplane/files/). It's only the Discussions that got moved to GitHub.

Or are you asking about something else?

**Reply by wordmuse** [reply]
That answers my question - thanks much!

Regards,
~ Bal
